Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -2080,7 +2081,7 @@ def update_remote_system_command(client: Client, args: Dict[str, Any], params: D
                 elif params.get('comment_tag') in tags:
                     key = 'comments'
                 user = entry.get('user', 'dbot')
-                text = f"({user}): {str(entry.get('contents', ''))}"
+                text = f"({user}): {str(entry.get('contents', ''))}\n\n Mirrored from Cortex XSOAR"
```

Comment:
please add test for this to verify it is sent as expected (with the text)

Reply:
Again, not sure I can test it.
It's important that it will send those requests from within this function.